fuzzers/005-tilegrid/add_tdb.py

```diff
@@ -111,6 +111,7 @@ def run(fn_in, fn_out, verbose=False):
         ("cfg_int", int_frames, int_words),
         ("monitor_int", int_frames, int_words),
         ("orphan_int_column", int_frames, int_words),
+        ("gtp_channel", 32, 20),
```

Comment:
The maximum range of words for each GTP_CHANNEL that I have seen is 22. I should double-check,but the math comes together with 22, as the first word of a GTP_COMMON tile, which is located after two GTP_CHANNELs is 44

Reply:
Changed to 22